src/lib/jsonParse/jsonParse.cpp

```diff
@@ -428,6 +432,13 @@ std::string jsonParse
   ptree              tree;
   ptree              subtree;
   std::string        path;
+  struct timespec    start;
+  struct timespec    end;
+
+  if (reqTimeStatistics)
```

Comment:
What about?

```
reqTimeStatistics && clock_gettime(CLOCK_REALTIME, &start);
```

(In general, as alternative to the multiple `if (reqTimeStatistics) {...}` structures that this PR introduces).

Not sure if that is a good idea, just to put the alternative on the table. First, because I haven't checked it it works :). Second, although it is more compact it could be considered less clear (for me is equivalently clear than the `if` but not sure others...).